src/main/java/org/dinosaur/foodbowl/domain/follow/repository/FollowRepository.java

```diff
@@ -1,8 +1,17 @@
 package org.dinosaur.foodbowl.domain.follow.repository;
 
+import java.util.List;
 import org.dinosaur.foodbowl.domain.follow.entity.Follow;
+import org.dinosaur.foodbowl.domain.member.entity.Member;
 import org.springframework.data.repository.Repository;
 
 public interface FollowRepository extends Repository<Follow, Long> {
 
+    List<Follow> findAll();
+
+    Follow save(Follow follow);
+
+    void deleteAllByFollowing(Member member);
```

Comment:
```suggestion
    void deleteAllByFollowing(Member following);
```
파리미터 네이밍도 follwing으로 구분해주시는 건 어떠신가요? 🙃

Reply:
좋습니다 👍